src/components/CommentList.jsx

```diff
@@ -39,7 +39,11 @@ export default function CommentList({ review_id, username }) {
           {activeComments.map((comment) => {
             return (
               <li key={comment.comment_id}>
-                <CommentCard {...comment} />
+                <CommentCard
+                  {...comment}
+                  activeComments={activeComments}
```

Comment:
I believe the CommentCard doesn't use `activeComments`, only `setActiveComments`